src/native/public/mono/metadata/details/class-functions.h

```diff
@@ -131,6 +131,8 @@ MONO_API_FUNCTION(MONO_RT_EXTERNAL_ONLY mono_bool, mono_class_is_delegate, (Mono
 
 MONO_API_FUNCTION(MONO_RT_EXTERNAL_ONLY mono_bool, mono_class_implements_interface, (MonoClass* klass, MonoClass* iface))
 
+MONO_API_FUNCTION(void, mono_class_get_variance_search_table, (MonoClass *klass, MonoClass ***table, int *table_size));
```

Comment:
This doesn't need to be a public API.  Add the declaration to `class-internals.h` instead